auto-updater.js

```diff
@@ -1,6 +1,5 @@
 const { autoUpdater } = require('electron');
 const { version } = require('./package');
-const notify = require('./notify');
```

Comment:
I think you still need to require this module, since it adds listeners to the `app` object.